### PR TITLE
PLAT-115302: Sync up VirtualList

### PR DESCRIPTION
**Pull request by MikyungKim**
This PR syncs up agate/VirtualList with sandstone/VirtualList.
Some of bug fixes from sandstone/VirtualList needs to be synced to agate/VirtualList.
Please refer to the ticket for details.

Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

### CODE REVIEW

File: VirtualList/useThemeVirtualList.js [hunk 1]
```diff
@@ -350,32 +331,25 @@ const useThemeVirtualList = (props) => {
...
 	delete rest.focusableScrollbar;
-	delete rest.scrollContainerContainsDangerously;
 	delete rest.scrollAndFocusScrollbarButton;
+	delete rest.scrollContainerContainsDangerously;
+	delete rest.scrollContainerHandle;
 	delete rest.scrollContainerRef;
 	delete rest.scrollContentHandle;
 	delete rest.spotlightId;
-	delete rest.scrollContainerHandle;
+
```
Comment (jeonghee27) [inline VirtualList/useThemeVirtualList.js:348]: Why did you add an empty line?

**Comment by jeonghee27** [comment]
I think We need a plan to better sync by separating the parts for Overscroll, scrollbar, and aria after this ticket.
I think we should discuss it.

File: VirtualList/useThemeVirtualList.js (hunk 1 above)
Comment (MikyungKim) [reply VirtualList/useThemeVirtualList.js:348]: Oh, by mistake. I'll remove.

**Comment by MikyungKim** [comment]
About `noScrollByDrag`, I think we removed the prop from agate because it's determined by `platform.touchscreen` inside of useScroll. I'm wondering why sandstone has this prop...

About DOM structure, I think we kept this because overscroll effect of agate has not been determined and unlike sandstone, the content is resized based on the scrollbar's existence so we didn't just touch anything from the original one.
We can discuss it later.
